src/helm/benchmark/run_specs/audio_run_specs.py

```diff
@@ -0,0 +1,52 @@
+"""Run spec functions for audio scenarios."""
+
+from typing import List, Optional
+from helm.benchmark.adaptation.adapter_spec import (
+    AdapterSpec,
+)
+from helm.benchmark.adaptation.adapters.adapter_factory import ADAPT_GENERATION_MULTIMODAL
+from helm.benchmark.metrics.common_metric_specs import (
+    get_classification_metric_specs,
+    get_exact_match_metric_specs,
+)
+from helm.benchmark.run_spec import RunSpec, run_spec_function
+from helm.benchmark.scenarios.scenario import ScenarioSpec
+
+
+def _get_multimodal_generation_adapter_spec(
+    instructions: str = "",
+    max_train_instances: int = 0,
+    max_tokens: int = 5,
+    temperature: float = 0.01,
```

Comment:
Why is the default temperature 0.01?

Reply:
Sorry, this was a mistake; I changed it to 0.0 instead.